src/box/sql/select.c

```diff
@@ -374,6 +374,7 @@ sqlSelectNew(Parse * pParse,	/* Parsing context */
 		 Expr * pOffset)	/* OFFSET value.  NULL means no offset */
 {
 	Select standin;
+	memset(&standin, 0, sizeof(standin));
```

Comment:
If this isn't necessary, we shouldn't do that just to silence static analyzers. Please drop this patch.

Reply:
Dropped.